src/css/components/_img-content.scss

```diff
@@ -43,3 +43,38 @@
      }
   }
 }
+
+.afo {
+  h2 {
+    text-align: center;
+  }
+  p {
+    text-align: center;
+  }
```

Comment:
These entities should be classed. We have a `text--centered` that should be used whenever centering text.